### PR TITLE
Add API to get value/argument axes

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

File: js/viz/docs/docObjectsDxChart.js [hunk 1]
```diff
@@ -94,3 +94,25 @@ var chartPointObject = {
...
+/**
+* @name chartAxisObject
+* @publicName Axis
+* @type object
+*/
+var chartAxisObject = {
+    /**
+    * @name chartAxisObjectMethods.visualRange
+    * @publicName visualRange()
+    * @type function()
```
Comment (RomanTsukanov) [inline js/viz/docs/docObjectsDxChart.js]: type is unnecessary in both these methods

File: js/viz/docs/docObjectsDxChart.js [hunk 2]
```diff
@@ -94,3 +94,25 @@ var chartPointObject = {
...
+    * @returns Array<number,string,Date>
+    */
+    visualRange: function() { },
+    /**
+    * @name chartAxisObjectMethods.visualRange
+    * @type function(visualRange)
+    * @param1 visualRange:Array<number,string,Date>
+    * @publicName visualRange(visualRange)
```
Comment (RomanTsukanov) [inline js/viz/docs/docObjectsDxChart.js]: Please declare the publicName after the name for consistency.

File: js/viz/docs/doccharts.js [hunk 3]
```diff
@@ -1977,6 +1977,25 @@ var dxChart = {
     * @param2 endValue:Number|Date|string
     */
     zoomArgument: function() { },
+    /**
+    * @name dxchartmethods.getValueAxes
```
Comment (RomanTsukanov) [inline js/viz/docs/doccharts.js]: I thought we have agreed to name the methods with a singular "axis"

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — DISMISSED** [review]
Doccomments LGTM

**Review by roman-simionov — DISMISSED** [review]

File: js/viz/docs/docObjectsDxChart.js [hunk 4]
```diff
@@ -94,3 +106,23 @@ var chartPointObject = {
     */
     getBoundingRect: function() { }
 };
+
+/**
+* @name chartAxisObject
+* @publicName Axis
```
Comment (lukyanovas) [inline js/viz/docs/docObjectsDxChart.js]: Please, don't use publicName for this object.

Comment (alexanderPolosatov) [reply js/viz/docs/docObjectsDxChart.js]: Done.

**Review by roman-simionov — APPROVED** [review]

**Pull request merged**